src/Sylius/Behat/Page/Shop/Checkout/AddressingPage.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Checkout;
+
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Sylius\Behat\Page\SymfonyPage;
+use Sylius\Component\Core\Model\AddressInterface;
+use Symfony\Component\Intl\Intl;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class AddressingPage extends SymfonyPage implements AddressingPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return 'sylius_shop_checkout_addressing';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function chooseDifferentBillingAddress()
+    {
+        $billingAddressSwitch = $this->getElement('different_billing_address');
+        $this->assertCheckboxState($billingAddressSwitch, false);
+        $billingAddressSwitch->check();
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws ElementNotFoundException
+     */
+    public function checkValidationMessageFor($element, $message)
+    {
+        $foundElement = $this->getFieldElement($element);
+        if (null === $foundElement) {
+            throw new ElementNotFoundException($this->getSession(), 'Validation message', 'css', '.pointing');
+        }
+
+        return $message === $foundElement->find('css', '.pointing')->getText();
```

Comment:
This will throw `getText() on null` when no element will not be found. This method should be implemented the same as in [Crud\UpdatePage](https://github.com/michalmarcinkowski/Sylius/blob/21e7f0d95196ca97ad33ac5f7667be531fabf6aa/src/Sylius/Behat/Page/Admin/Crud/UpdatePage.php#L55).